### ISSUE TITLE
Update images for examples

### ISSUE THREAD

**Issue by lcy-seso** [PaddlePaddle/models#499]
Some images in the Paddle Models repo should be redrawn (There are directly from the Internet (we add the reference, however, this may not be enough.)) :
1.  https://github.com/PaddlePaddle/models/blob/develop/ctr/images/wide_deep.png
2. https://github.com/PaddlePaddle/models/blob/develop/mt_with_external_memory/image/lstm_c_state.png
3.  https://github.com/PaddlePaddle/models/blob/develop/mt_with_external_memory/image/turing_machine_cartoon.gif
4. images in this directory: https://github.com/PaddlePaddle/models/tree/develop/dssm/images
5. https://github.com/PaddlePaddle/models/blob/develop/scene_text_recognition/images/ctc.png
6. https://github.com/PaddlePaddle/models/blob/develop/scene_text_recognition/images/feature_vector.png
7. https://github.com/PaddlePaddle/models/blob/develop/scene_text_recognition/images/transcription.png

**Comment by luotao1** [comment]
请问这些图，是全部照着重画么？
- 重画过程中，需要把图中的英文改成中文么？
- 照着重画后，我们可以不加引用了么？

book中的所有图，都是风格配色统一的。models下的图也需要这种标准么？

**Comment by lcy-seso** [comment]
我个人的看法，不希望未来开发一个models下应用对开发者来说成本太高。上面列出来的几幅图最主要的问题是直接使用了从网上/论文中的原图，我们尽量都加上了引用，但一些博客图片现在没有引用，潜在有一定的风险。

1. 上面列出来的图照着原图画就可以；
1. 图片使用英文即可；

- models下面的图片如果统一风格锦上添花，美观清晰，是挺好的事情。
- 但我们很难让未来models的作者添加同样风格的图片（他们大多是程序员），基于这样的考虑，我觉得即使是现在的models不必追求全部严格的风格统一。以准确、简洁、清晰、美观为目标即可。

**Comment by luotao1** [comment]
好的。上面的图中，有一张重画难度太大，这张还是加引用么？ https://github.com/PaddlePaddle/models/blob/develop/mt_with_external_memory/image/turing_machine_cartoon.gif

**Comment by lcy-seso** [comment]
抱歉，没有说明这张图的问题。这张图之前的问题比较大（很难确认原作者），但不需要重画成一样的，可以按照下面这张的样子去重绘，图的 title 可以不变：

![image](https://user-images.githubusercontent.com/5842774/33356229-558571ae-d4f6-11e7-94ed-df43795b4f64.png)

**Comment by jetfuel** [comment]
mt_with_external_memory 跟 scene_text 有關的文章還沒有發布，我們會晚一點才動到他們。目前會集中在 dssm 跟 ctr 有關的圖

**Comment by lcy-seso** [comment]
谢谢你们~

**Comment by varunarora** [comment]
So I just wanted to provide some information which would be useful to understand whether or not we can use images from the Internet on our models (and/or book pages). Note that I am not a lawyer and my notes are only based on the US jurisdiction.

**TL;DR**: It’s not black-or-white; every case is different so just have to be careful to not “rip-off” large ideas in papers and presentations without adequate thought and credits.

—

In the US, the rights of creators of ideas and content is governed by the Copyright law. In its essence, it protects the rights or original ideas that have been recorded and shared. The basics of this are easily Google-able; its a big concept so I am not going to cover it here.

There is a clause in the copyright law that allows people to adopt / re-use original work. This is called the **Fair Use act**. You can read about Fair Use here: https://www.copyright.gov/fair-use/more-info.html.

In essence, there are some criteria to determine whether the use of somebody else’s work is fair or not (this is what judges in courts think about, if it comes down to that). These include (with an attempted trivialized explanation):

1. **Non-commercial non-profit use**: It is ideal if the work is for education and non-commercial purposes. It is ideal If the work has been transformed in a substantial way.

2. **Nature of work**: Your violation or ability to re-use depends on how creative or original some work is.

3. **Amount of work used**: Basically, if you use a large proportion of work, your more likely to be screwed.

4. **Potential market for the value of the work**: If you are harming the future value of the original work by your use, it’s not cool.

__

### My thoughts regarding the images in question above

  - Some of the images come from papers licensed by professional society publications. Re-publication in substantial ways requires permission from the publications, and the the authors. Some of them require a payment of a fee.

  - If we aren’t planning to transform them significantly to adopt their ideas into something else, but rather just using them to explain our implementation (or a deviation from their work), redoing them doesn’t have much advantages. In fact, re-doing them will make us feel like it is okay to reduce how much we credit them down the line, and that is something we should prevent.

  - Building on the previous point, credit them adequately and respectfully.

  - If we really have a transformation on the images, that drives home our points or removes unnecessary details that are distracting, we may redo the images but ALSO still credit the original work.

  - Use only pieces that we genuinely need. Even if that means parts of larger graphics that are most useful to the point we are driving forward.


So essentially if you are using small amounts of the original work in respectful ways that will not hurt / offend the original creator and are crediting them adequately, there may be no need to re-do the images.

**Comment by shanyi15** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持! 
 Hello, this issue has not been updated in the past month. We will close it today for the sake of other user‘s experience. If you still need to follow up on this question after closing, please feel free to reopen it. In that case, we will get back to you within 24 hours. We apologize for the inconvenience caused by the closure and thank you so much for your support of PaddlePaddle Group!
